app/javascript/articles/components/ContentTitle.jsx

```diff
@@ -1,43 +1,39 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
-
 import { articlePropTypes } from '../../src/components/common-prop-types';
 
-export const ContentTitle = ({ article, currentTag }) => (
-  <h3>
-    {article.flare_tag && currentTag !== article.flare_tag.name && (
-      <span
-        className="tag-identifier"
-        style={{
-          background: article.flare_tag.bg_color_hex,
-          color: article.flare_tag.text_color_hex,
-        }}
-      >
-        {`#${article.flare_tag.name}`}
-      </span>
-    )}
-    {article.class_name === 'PodcastEpisode' && (
-      <span className="tag-identifier">podcast</span>
-    )}
-    {article.class_name === 'User' && (
-      <span
-        className="tag-identifier"
-        style={{ background: '#5874d9', color: 'white' }}
-      >
-        person
-      </span>
-    )}
-    {filterXSS(article.title)}
-  </h3>
+export const ContentTitle = ({ article }) => (
+  <h2 className="crayons-story__title">
+    <a href={article.path} id={`article-link-${article.id}`} >
+      {/* {article.flare_tag && currentTag !== article.flare_tag.name && (
+        <span
+          className="tag-identifier"
+          style={{
+            background: article.flare_tag.bg_color_hex,
+            color: article.flare_tag.text_color_hex,
+          }}
+        >
+          {`#${article.flare_tag.name}`}
+        </span>
+      )}
+      {article.class_name === 'PodcastEpisode' && (
+        <span className="tag-identifier">podcast</span>
+      )}
+      {article.class_name === 'User' && (
+        <span
+          className="tag-identifier"
+          style={{ background: '#5874d9', color: 'white' }}
+        >
+          person
+        </span>
+      )} */}
```

Comment:
We don't want to have these promoted tags here, in the title. Instead it would be nice to have them in TagList.jsx.. Example:

![image](https://user-images.githubusercontent.com/108287/78777553-0b50f180-799a-11ea-99fa-3d8fe3d64bd2.png)